azurelinuxagent/common/osutil/debian.py

```diff
@@ -61,10 +61,10 @@ def get_dhcp_lease_endpoint(self):
         return self.get_endpoint_from_leases_path('/var/lib/dhcp/dhclient.*.leases')
 
 
-class DebianOS8Util(DebianOSUtil):
+class DebianOSModernUtil(DebianOSBaseUtil):
 
     def __init__(self):
-        super(DebianOS8Util, self).__init__()
+        super(DebianOSModernUtil, self).__init__()
```

Comment:
Same problem: this is terrible use of `super()`. Try this:
```
class DebianOSModernUtil(DebianOSBaseUtil):
    def __init__(self):
        DebianOSBaseUtil.__init__()
        self.jit_enabled = True
        self.service_name = self.get_service_name()
```

Reply:
Let's push this issue to another unit of work to keep this PR focused on gen2 resource disk.